src/generate-analysis.ts

```diff
@@ -235,6 +250,18 @@ function serializeElementMixin(
   return metadata;
 }
 
+function serializePolymerBehaviorAsElementMixin(
+    behavior: ResolvedPolymerBehavior, packagePath: string): ElementMixin {
+  const metadata = serializeElementLike(behavior, packagePath) as ElementMixin;
+  metadata.name = behavior.className;
+  metadata.privacy = behavior.privacy;
+  metadata.metadata.isPolymerBehavior = true;
```

Comment:
prefer:

```javascript
metadata.metadata = {polymer: {isBehavior: true}};
```

The idea is that an analysis object should be able to host multiple metadata sections contributed by multiple frameworks/libraries